torch/csrc/jit/codegen/cuda/scheduler/reduction_utils.cpp

```diff
@@ -514,7 +536,7 @@ void multiReductionInliner(
         compute_from,
         compute_to,
         -1,
-        ComputeAtMode::MostInlined,
+        ComputeAtMode::BestEffort,
```

Comment:
Not really clear what the motivation of this change is.

Reply:
There shouldn't be any reductions present in that section so it shouldn't matter. Maybe we should use the least aggressive mode that fits the situation, but honestly don't know why I changed it. Shouldn't matter.